guide/src/README.md

```diff
@@ -1,13 +1,22 @@
 # Introduction
 
-> cargo, make me a project
+```rust
+cargo generate /me/a-project
```

Comment:
I'm not so sure that this slogan should be changed within this PR. Seems pretty unrelated to me.